cypress/integration/brewery/TableParameters-invalid_files.spec.js

```diff
@@ -8,6 +8,67 @@ import { DEFAULT_SCENARIOS_LIST } from '../../fixtures/stubbing/default';
 
 const CSV_INVALID_FILE_PATH = 'customers_invalid.csv';
 const XLSX_INVALID_FILE_PATH = 'customers_invalid.xlsx';
+const CSV_TOO_MUCH_PARAMETERS = 'customers_too_much_parameters.csv';
+const XLSX_TOO_MUCH_PARAMETERS = 'customers_too_much_parameters.xlsx';
+const CSV_ONLY_NAME_PARAMETER = 'customers_only_name_parameter.csv';
+const XLSX_ONLY_NAME_PARAMETER = 'customers_only_name_parameter.xlsx';
```

Comment:
Typo: it should be "too many" instead of "too much" :slightly_smiling_face: 
Also, could we replace "parameter" by "column" or "field" in the names of the files and constants?